web/docs/guides/api.mdx

```diff
@@ -216,6 +216,21 @@ curl '[SUPABASE_URL]/rest/v1/todos' \
 [delete()](/docs/reference/javascript/delete),
 [rpc()](/docs/reference/javascript/rpc) (call Postgres functions).
 
+### Safeguards towards accidental deletes and updates
```

Comment:
@MildTomato do we want to mention this somewhere within the dashboard itself as well?